scripts/redis

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 set -e
-CONTAINER_NAME=hintr_redis
+CONTAINER_NAME=hintr-redis
```

Comment:
Changing name here so that we can run up the front end dependencies at the same time as we run this. If we want to work on both at the same time.